packages/vsce/src/api/SshMvsApi.ts

```diff
@@ -77,7 +77,7 @@ export class SshMvsApi extends SshCommonApi implements MainframeInteraction.IMvs
             stream: options.file ? writeStream : undefined,
         });
         if (options.stream != null) {
-            options.stream.write(B64String.decode(response.data));
+            options.stream.write(B64String.decode(response.data!));
```

Comment:
Not a request for changes, just an observation - I think we've had some discussions before about making the `data` property on our responses use implicit null, to avoid requiring the `!` operator.

I've done that by setting the `tstype` to required in https://github.com/zowe/zowe-native-proto/pull/516/commits/8fdadc7c19b05a8cc1cd64870a42f233f9c1eaf6#diff-f522cefe51c3a6c6e1fa68d3ac3ee3a3519a7c2920a3da516876a126539b4ef6

If we prefer to keep the `data` property optional, happy to revert the change in that PR 😋